src/main/kotlin/network/warzone/mars/player/listeners/DismountListener.kt

```diff
@@ -0,0 +1,58 @@
+package network.warzone.mars.player.listeners
+
+import network.warzone.mars.Mars
+import network.warzone.mars.feature.ResourceType
+import network.warzone.mars.player.controllers.SitController
+import org.bukkit.Bukkit
+import org.bukkit.Location
+import org.bukkit.craftbukkit.v1_8_R3.entity.CraftArmorStand
```

Comment:
This PR would break compatibility for modern server versions since you are directly using craftbukkit imports in some files.
Currently Mars does not have any utility functions for handling NMS/OBC imports so that would need to be implemented as well